app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -807,15 +790,25 @@ impl Model {
         header_pos: f32,
     ) -> Vector2 {
         if is_header_selected {
-            Vector2(0.0, size.y / 2.0 - header_geometry.height / 2.0 - header_pos)
+            let header_y = self.header_y(header_geometry, size, header_pos);
+            Vector2(0.0, header_y)
         } else {
-            let max_selection_pos_y = size.y / 2.0 - list_view::entry::HEIGHT - header_pos;
-            let selection_pos_y = entries_selection_position.y.min(max_selection_pos_y);
-            let selection_pos = Vector2(entries_selection_position.x, selection_pos_y);
-            self.entries.position().xy() + selection_pos
+            self.entries.position().xy() + entries_selection_position
         }
     }
 
+    /// The y position of the header. The position is restricted by the borders of the
+    /// component group.
```

Comment:
Again, this doc comment does not really explains what this function does. It tells that it is "y position of the header", but this function contains a few lines of code that do something. But what they really do is not explained in the docs.

Reply:
Rephrased, though I don't think the documentation must describe every implementation detail of every method. It is enough to know that this particular method returns a y position of the header, and this position is restricted by the borders of the component group.